StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/DocumentationResources.de-DE.resx

```diff
@@ -129,6 +129,9 @@
   <data name="NonPrivateConstructorStandardTextSecondPart" xml:space="preserve">
     <value> {0}</value>
   </data>
+  <data name="ParameterNotUsed" xml:space="preserve">
+    <value>Dieser Parameter wird nicht verwendet.</value>
```

Comment:
💡 I recommend updating your GitHub profile to include your name. Right now we just see **dfyx**.